NerdyNomads/Text-Savvy
# Add SideBar Component

## What does this PR do?
- Add Sidebar to App.js
  - Added a dashboard component to contain the text list
  - Note: Manage Account section is TBA (currently shows an alert modal with a temporary message)

### Screenshots (optional)

![Screenshot 2022-03-03 214425](https://user-images.githubusercontent.com/24909769/156695591-e3a1613d-a27f-4821-8e1a-dc17316ec73e.png)

### Steps to test/review (optional)
- Run front-end
  -Make sure: your name (email without domain) should show up, no console log errors, you can logout seamlesslly.
- resize window to test responsiveness

## Checklist before merging
      - Reviewers
      - Assignee
      - Labels
      - Project
      - Milestone
      - Linked Issues
